docs/xcm/building-with-xcm/xc-reserve-transfer.md

```diff
@@ -16,7 +16,11 @@ Suppose we want Shibuya `SBY` tokens to be present on Shiden as a wrapped asset
 During the actual transfer the following happens:
 1. Some `SBY`s are moved from source account to the sovereign account of Shiden on Shibuya
 2. An XCM containing `ReserveTransferAssets` instruction is sent to Shiden
+<<<<<<< HEAD
```

Comment:
Merge conflict remnants, should be cleaned up.

Reply:
Ouch